account_invoice_inter_company/models/account_invoice.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models, _
+from odoo.exceptions import Warning as UserError
+from odoo.tools import float_compare
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    auto_generated = fields.Boolean(string='Auto Generated Document',
+                                    copy=False, default=False)
```

Comment:
seems better to add help attribute in all fields #usability

Reply:
Hi, David,
Thank you for the review but I think it's useful to add the help attribute for fields that are visible in a view. This is not the case for the 'auto_generated' field.